_locales/ru/messages.json

```
@@ -1,4 +1,4 @@
-{
+\{
```

JSON structure is broken here.
```suggestion
{
```